demos/universal-sentence-encoder/README.md

```diff
@@ -50,17 +51,18 @@ The layer SentencepieceTokenizer expects on the input a list of strings. The CPU
 
 Until it is published, the docker image with OpenVINO Model Server including the CPU extension has to be built using the commands:
 
-```
+```bash
 git clone -b develop https://github.com/openvinotoolkit/model_server
 cd model_server
-make docker_build SENTENCEPIECE=1 OV_SOURCE_ORG=rkazants OV_CONTRIB_ORG=rkazants OV_SOURCE_BRANCH=muse_perf OV_CONTRIB_BRANCH=rkazants/sentence_tokenizer_optimization OV_USE_BINARY=0
+make docker_build SENTENCEPIECE=1 OV_CONTRIB_ORG=rkazants OV_CONTRIB_BRANCH=rkazants/sentence_tokenizer_optimization OV_USE_BINARY=0
+cd ..
```

Comment:
Why are we moving out of model_server repo?

Reply:
this it needed to be again in the same folder with the model folder
Later the section about building the image will be dropped and public image will be used